packages/ipfs-core-config/README.md

```diff
@@ -1,8 +1,7 @@
 # ipfs-core-utils <!-- omit in toc -->
 
-[![](https://img.shields.io/badge/made%20by-Protocol%20Labs-blue.svg?style=flat-square)](http://protocol.ai)
-[![](https://img.shields.io/badge/project-IPFS-blue.svg?style=flat-square)](http://ipfs.io/)
-[![](https://img.shields.io/badge/freenode-%23ipfs-blue.svg?style=flat-square)](http://webchat.freenode.net/?channels=%23ipfs)
+[![](https://img.shields.io/badge/made%20by-Protocol%20Labs-blue.svg?style=flat-square)](https://protocol.ai)
```

Comment:
Could you PR https://github.com/ipfs/aegir/blob/master/src/check-project/readme/header.js#L12-L18 with these changes as well please?